frontend/awx/access/credentials/CredentialForm.tsx

```diff
@@ -518,6 +601,22 @@ function CredentialSubForm({
               />
             );
           } else {
+            if (isEncryptedValues) {
+              if (field.secret && isEditMode && isEncryptedValues[field.id] === true) {
+                //console.log(field.id, 'is an encrypted field');
+                return (
+                  <SingleLineEncryptedInput
```

Comment:
What is the reason behind wrapping this wrapper within another one: SingleLineEncryptedInput -> EncryptedTextFormInput->CredentialTextInput